### PR TITLE
NEWS.md: update notes for 0.7.0

### PR DESCRIPTION
**Pull request by johanmalm**
Needs a final tweak prior to releasing `0.7.0` (e.g. date, line-count and so on)

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -25,6 +26,60 @@ The format is based on [Keep a Changelog]
...
+- Teach MoveToEdge to move windows to adjacent outputs.
+  Written-by: @ahesford
+- Implement `<font place="InactiveWindow">`. Written-by: @ludg1e (#1292)
+- Implement cursor-shape-v1 protocol to allow Wayland clients to request a
+  buffer for a cursor shape from a compositor. Written-by: @heroin-moose
+- Implement fractional-scale-v1 protocol to allow Wayland clients to properly
+  scale on outputs with fractional scale factor. Written-by: @heroin-moose
+- Add ResizeTo action (#1261)
+- Allow going backwards in window-switcher OSD by using arrow-up or arrow-left.
+  Written-by: @jg6766
```
Comment (jp7677) [inline NEWS.md:50]: Thanks, though I would prefer @jp7677 here :)

File: NEWS.md [hunk 2]
```diff
@@ -25,6 +26,60 @@ The format is based on [Keep a Changelog]
...
+  fix sfwbar minimize action. (#1342)
+- Move layer-shell popups from the background layer to the top layer to render
+  the above normal windows. Previously this was only done for the bottom
+  layer. In support of Raspberry Pi's `pcmanfm --desktop`. (#1293)
+- Calculate `usable_area` before positioning clients to ensure it is correct
+  before non exclusive-zone layer-shell clients are positioned or resized.
+  (#1285)
+- Prevent overriding XWayland maximized/fullscreen/tiled geometry to fixes an
+  issue where some XWayland views (example: xfce4-terminal) do not end up with
+  exactly the correct geometry when tiled.
```
Comment (Consolatis) [inline NEWS.md:72]: `s/to fixes/to fix/`

File: NEWS.md (hunk 1 above)
Comment (johanmalm) [reply NEWS.md]: Apologies. I'm not sure what made we write jg6766... Now fixed.

File: NEWS.md (hunk 2 above)
Comment (johanmalm) [reply NEWS.md]: Thanks. Done.

File: NEWS.md [hunk 3]
```diff
@@ -25,6 +26,60 @@ The format is based on [Keep a Changelog]
...
+  by Qt to determine areas of the screen that popup menus should not overlap.
+
+# Fixed
+
+- Fix issue with XWayland surfaces completely offscreen not generating commit
+  events and therefore preventing them from moving onscreen.
+- Do not de-active windows when layer-shell client takes keyboard focus, to
+  fix sfwbar minimize action. (#1342)
+- Move layer-shell popups from the background layer to the top layer to render
+  the above normal windows. Previously this was only done for the bottom
```
Comment (Consolatis) [inline NEWS.md:68]: to render them (missing `m` for `them`)

**Comment by Consolatis** [comment]
Should we add something regarding 0.6 branch going into maintenance mode? Or is that better suited for the release notes / maintainer notification?

**Comment by johanmalm** [comment]
> Should we add something regarding the 0.6 branch going into maintenance mode? Or is that better suited for the release notes / maintainer notification?
> 
> Edit: Forgot to mention, with the layer-shell fix above this LGTM.

Thanks. Good idea. I'll add "A `0.6` branch will be created, should bug fixes be required "

**Comment by Consolatis** [comment]
> I'll add "A `0.6` branch will be created, should bug fixes be required "

I've marked some things with a backport label: https://github.com/labwc/labwc/issues?q=label%3Abackport

**Pull request merged**